SCClassLibrary/Common/Audio/UGen.sc

```diff
@@ -576,8 +576,8 @@ MultiOutUGen : UGen {
 	}
 
 	initOutputs { arg numChannels, rate;
-		if(numChannels.isNil or: { numChannels < 1 }, {
-			Error("%: wrong number of channels (%)".format(this, numChannels)).throw
+		if(numChannels.isInteger.not or: { numChannels < 1 }, {
+			Error("%: numChannels must be nonzero positive integer (%)".format(this, numChannels)).throw
```

Comment:
..."must be a nonzero positive integer."

I tend to prefer that the passed argument value is next to the arg name in the message. What do you think?
```
"%: numChannels (%) must be a nonzero positive integer.".format(this, numChannels)`
>>>
ThisUGen: numChannels (nil) must be a nonzero positive integer.
```
or even more explicitly
```
"%: numChannels must be a nonzero positive integer, but received (%).".format(this, numChannels)
>>>
ThisUGen: numChannels must be a nonzero positive integer, but received nil.
```